q2_demux/plugin_setup.py

```diff
@@ -135,7 +138,9 @@
               'average positional qualities across all of the sequences '
               'selected. If input sequences are paired end, plots will be '
               'generated for both forward and reverse reads for the same `n` '
-              'sequences.')
+              'sequences.'),
+        'delta': ('The acceptable delta cutoff for disparities in sequence '
+                  'length')
```

Comment:
I'm still on the fence about this `delta` feature, it feels a little overkill since the visualization just displays a warning if truncation happens. It seems like the parameter only exists to help users silence the warning message, but if they know enough to silence the warning, I think we can assume they understood the warning message in the first place.

I'd be in favor of removing this parameter altogether, what do you guys think? It's also less code to test and maintain in the future. pinging @thermokarst